tests/kernel/sched/deadline/src/main.c

```diff
@@ -8,9 +8,13 @@
 #include <random/rand32.h>
 
 #define NUM_THREADS 8
-#define STACK_SIZE (256 + CONFIG_TEST_EXTRA_STACKSIZE)
+/* this should be large enough for us
+ * to print a failing assert if necessary
+ */
```

Comment:
Nitpicking: I'd skip the comment.  It's fair to assume that the code author thought the requested stack size was large enough :)

Reply:
Fair enough.

The comment was intended mainly to be a reminder that the stack size shouldn't just be large enough in the _passing_ case (it is), but also needs to be large enough to handle printing in the _failing_ case too (it wasn't).